base/applications/charmap/map.c

```diff
@@ -260,6 +260,7 @@ SetFont(PMAP infoPtr,
         {
             if (out[i] != 0xffff)
             {
+                if (ch[i] == 0x0020) continue;  // skip blank space character
```

Comment:
`' '` Would be more readable (and not require a comment?)

Reply:
Sure, that sounds good.  I included the full 0x0020 to identify ch[i] as a 16-bit value, but the implicit cast from char would definitely be more readable.  What's the recommended way of modifying this in the PR?  Should I just add a new commit?  Or would you like me to rebase and modify this commit only to remove multiple modifications from this line?

Reply:
If you're aware of git tricks for rebase and force push with lease, it's the best way to do it.

Reply:
Perfect, sounds good!  I've made that modification and pushed it.

Reply:
Ahhh cool!  I had never heard of force-with-lease.  I'll have to keep that in mind for next time I try something like this out.